src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { InvalidPage } from './components/InvalidPage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":personSlug" />
```

Comment:
The nested route with ':personSlug' seems to be missing an element. You should specify a component that will be rendered when navigating to '/people/:personSlug'.

Reply:
There is no point in adding the same element, because it does not change with the parent, and I use a nested route, because since version 6.* the ability to use the optional path parameterrh has been removed